app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -430,14 +432,19 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     showSnackbar(mActivity, R.string.text_file_selected);
                     String fileName = mFileUtils.getFileName(mTextFileUri);
                     if (fileName != null) {
-                        if (fileName.endsWith(Constants.textExtension)) mFileExtension = Constants.textExtension;
-                        else if (fileName.endsWith(Constants.docxExtension)) mFileExtension = Constants.docxExtension;
-                        else if (fileName.endsWith(Constants.docExtension)) mFileExtension = Constants.docExtension;
+                        if (fileName.endsWith(Constants.textExtension))
+                            mFileExtension = Constants.textExtension;
+                        else if (fileName.endsWith(Constants.docxExtension))
+                            mFileExtension = Constants.docxExtension;
+                        else if (fileName.endsWith(Constants.docExtension))
+                            mFileExtension = Constants.docExtension;
                         else {
                             showSnackbar(mActivity, R.string.extension_not_supported);
                             return;
                         }
                     }
+                    String nameWithoutExtension = mFileUtils.stripExtension(fileName);
```

Comment:
Is there a need to have this variable `nameWithoutExtension`
you can skip it and directly create mSelectedFileName